packages/esm-patient-chart-app/src/index.ts

```diff
@@ -29,19 +30,20 @@ function setupOpenMRS() {
   defineExtensionConfigSchema('nav-group', genericNavGroupConfigSchema);
   defineExtensionConfigSchema('dashboard', genericDashboardConfigSchema);
 
+  const t = i18next.getFixedT(null, moduleName);
   registerBreadcrumbs([
     {
       path: spaBasePath,
-      title: 'Patient',
+      title: () => Promise.resolve(t('patientBreadcrumb', 'Patient')),
```

Comment:
We shouldn't need the `Promise.resolve()` here, right? It's generally better to avoid wrapping things in promises if we can. (Also applies below).